packages/framer-scroller/components/ScrollerButton.tsx

```diff
@@ -27,9 +27,16 @@ export const ScrollerButton = m(
       ...buttonProps
     } = props
 
+    const isScrolling = useRef(false)
     const { getSnapPosition, scroll } = useScrollerContext()
     const scrollTo = useScrollTo()
-    const handleClick = () => scrollTo(getSnapPosition(direction))
+    const handleClick = async () => {
+      if (!isScrolling.current) {
```

Comment:
I think we already have a `scroll.animating`. You can use that variable to catch the "spammed" click